<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="new_file_Action"/>
    <addaction name="open_file_Action"/>
    <addaction name="save_file_Action"/>
    <addaction name="save_another_file_Action"/>
   </widget>
   <addaction name="menu_F"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="new_file_Action">
   <property name="text">
    <string>new file&amp;(ctrl + N)</string>
   </property>
  </action>
  <action name="save_file_Action">
   <property name="text">
    <string>save file&amp;(ctrl + S)</string>
   </property>
  </action>
  <action name="open_file_Action">
   <property name="text">
    <string>open file&amp;(ctrl + O)</string>
   </property>
  </action>
  <action name="save_another_file_Action">
   <property name="text">
    <string>save another file&amp;(ctrl + shift + S)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
